.pfnci/common.sh

```diff
@@ -12,20 +12,27 @@ gcloud auth configure-docker
 TEMP=$(mktemp -d)
 mount -t tmpfs tmpfs ${TEMP}/ -o size=100%
 
+get_local_version() {
+    git merge-base --is-ancestor HEAD v6 && LOCAL_VERSION=stable 
```

Comment:
https://git-scm.com/docs/git-merge-base says:

> --is-ancestor
> Check if the first <commit> is an ancestor of the second <commit>, and exit with status 0 if true, or with status 1 if not. Errors are signaled by a non-zero status that is not 1.


```suggestion
    git merge-base --is-ancestor v6 HEAD && LOCAL_VERSION=stable 
```